pootle/static/js/editor/mt/providers/CaighdeanTranslate.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) Pootle contributors.
+ *
+ * This file is a part of the Pootle project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import MTProvider from '../MTProvider';
+
+
+class CaighdeanTranslate extends MTProvider {
+
+  constructor(apiKey) {
+    super({
+      apiKey,
+      name: 'caighdean-translate',
+      displayName: 'Caighdean Translate',
+      url: "http://borel.slu.edu/cgi-bin/seirbhis3.cgi",
+	supportedLanguages: ['gd', 'gv', 'ga_IE', 'ga-IE'],
```

Comment:
This is not properly indented. Probably this and other indents are causing Travis to fail.